js/skin.js

```diff
@@ -136,13 +136,16 @@ var Skin = createReactClass({
    * @public
    * @description the function is called when we stop the rotation
    */
-  handleVrPlayerMouseUp: function() {
+  handleVrPlayerMouseUp: function() {    
     if (this.props.controller && this.props.controller.isVrStereo) {
       return;
     }
+
     if (this.props.controller && this.props.controller.videoVr) {
-      var isVrMouseMove = this.state.isVrMouseMove;
-      if (Utils.isIos()) {
+      var isVrMouseMove = this.state.isVrMouseMove,
+          isTouchEnd = typeof arguments[0] === 'object' && arguments[0].type === 'touchend';
+
```

Comment:
Could you add "var" for second variable instead of list of variable (according by our rules in eslint)?

Reply:
Could we use only isTouchEnd instead of Utils.isIos()?

Reply:
Could you add tests for the changes?

Reply:
It's risky to use only isTouchEnd instead of Utils.isIos(). Something can broke.
Tests already exists.